### DISCUSSION TITLE
Screenshot title

### DISCUSSION THREAD

**Discussion by radoslavirha** [LironEr/cypress-mochawesome-reporter#136] [Ideas]
Hi,

by default all the screenshots in report have title 'Screenshot'. Is it possible to customize it somehow?

Thanks

**Comment by LironEr** [comment]
Currently, it is not possible to change it.
I'm unsure if it will be possible, but I might change the title to the screenshot file name.
So if you take a screenshot, you will see "custom name" in the HTML report file.
```
cy.screenshot('custom-name');
```

Would it help?
Would you like other behavior?

**Reply by radoslavirha** [reply]
Hi @LironEr
I think it would be better :)

Thanks!

**Reply by LironEr** [reply]
#145

**Reply by LironEr** [reply]
Released in [v3.5.0](https://github.com/LironEr/cypress-mochawesome-reporter/releases/tag/v3.5.0)
